subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/CreateSwiftBundle.java

```diff
@@ -63,35 +68,42 @@ public void execute(CopySpec copySpec) {
                     }
                 });
 
-                copySpec.from(getInformationFile(), new Action<CopySpec>() {
-                    @Override
-                    public void execute(CopySpec copySpec) {
-                        copySpec.into("Contents");
-                    }
-                });
-
                 copySpec.into(getOutputDir());
             }
         });
 
+        File inputFile = getInformationFileIfExists();
+        File outputFile = new File(getOutputDir().getAsFile().get(), "Contents/Info.plist");
```

Comment:
Hrm... is `getOutputDir()` a file or a directory?  It seems to be a file on line 47 and in SwiftBasePlugin, but it seems to be used as a dir here.

Reply:
See the other comment on this issue.